src/app/mi-details/view-story/view-story.page.html

```diff
@@ -0,0 +1,145 @@
+      </div>
+      }
+    </div>
+  </div>
+
+@if(viewProjectDetails.story.summary){
+  <div class="summary-container">
+    <div class="summary-head">
+      <h1 class="summary-title">{{"SUMMARY" | translate}}</h1>
+      <button class="summary-copy" (click)="copyText()">
+        <ion-icon name="copy-outline"></ion-icon>
+      </button>
+    </div>
+    <p class="summary-text">{{viewProjectDetails.story.summary}}</p>
+  </div>
+}
+  
+  @if(viewProjectDetails.story.problemStatement){
+    <app-mi-details-card
+        [cardConfig]="{ title: 'PROBLEM_STATEMENT', text: viewProjectDetails.story.problemStatement }">
+    </app-mi-details-card>
+  }
+
+
+@if(viewProjectDetails.story.objective){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'OBJECTIVE', text: viewProjectDetails.story.objective }">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.story.timeline){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'TIMELINE', text: viewProjectDetails.story.timeline}">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.tasks.length>0){
+  <app-mi-details-card
+    [cardConfig]="{ title: 'ACTION_STEPS', tasks: viewProjectDetails.tasks }">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.story.impact){
+  <app-mi-details-card 
+    [cardConfig]="{ title: 'IMPACT', text: viewProjectDetails.story.impact }">
+  </app-mi-details-card>
+}
+
+@if(viewProjectDetails.attachments.length > 0){
+  <div class="resource-container">
+    <h1 class="resource-title">
+      <ion-icon name="star"></ion-icon>
+      {{"RESOURCES" | translate}}
+    </h1>
+    @if(resource.images.length > 0){
+      <div class="resource-item">
+        @for(image of resource.images;track image){
+          <div class="item">
+            <ion-icon name="image-outline"></ion-icon>
+            <a [href]="image.url" download="{{image.url}}">
+              <span>{{image.name}}</span>
+            </a>
+          </div>
+        }
+      </div>
+    }
+    @if(resource.documents.length > 0){
+      <ul class="resource-list">
+        @for(document of resource.documents;track document){
+        <li>
+          <a [href]="document.url" download="{{document.url}}">
+            <ion-icon name="link"></ion-icon>
+            {{document.name}}
+          </a>
+        </li>
+      }
+    </ul>
+    }
+
+    @if(resource.videos.length > 0){
+      <div class="resource-item">
+        @for(video of resource.videos;track video){
+          <div class="item">
+            <ion-icon name="videocam"></ion-icon>
+            <a [href]="video.url" download="{{video.url}}">
+              <span>{{video.name}}</span>
+            </a>
+          </div>
+        }
+      </div>
+    }
+  </div>
+}
+
+<div class="bottom-container">
+    @if(viewProjectDetails.story.text){
+      <p class="bottom-text">{{viewProjectDetails.story.text}}</p>
+    }
+    @if(viewProjectDetails.attachments.length > 0){
+    <div class="evidences-container">
```

Comment:
I think for resources also u are using the same array right. Then why are we showing the same thing in the Evidence section?